<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NODE319" name="com.sharepoint.TestNGtest.Annotations" tests="5" failures="1" timestamp="14 Nov 2014 09:50:39 GMT" time="0.020" errors="0">
  <testcase name="happen" time="0.000" classname="com.sharepoint.TestNGtest.Annotations"/>
  <testcase name="happen" time="0.010" classname="com.sharepoint.TestNGtest.Annotations">
    <failure type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- happen -->
  <testcase name="happen" time="0.000" classname="com.sharepoint.TestNGtest.Annotations"/>
  <testcase name="happen" time="0.000" classname="com.sharepoint.TestNGtest.Annotations"/>
  <testcase name="happen" time="0.010" classname="com.sharepoint.TestNGtest.Annotations"/>
</testsuite> <!-- com.sharepoint.TestNGtest.Annotations -->
